charts/spire/charts/spire-server/values.yaml

```diff
@@ -220,23 +218,21 @@ nodeAttestor:
 
 # tornjak - Tornjak specific configuration
 tornjak:
+  # -- Deploys Tornjak API (backend)
   enabled: false
-  # image - Tornjak image (frontend + backend) if not separated above
-  image: # ghcr.io/spiffe/tornjak
+  # -- Tornjak API image
+  image:
     registry: ghcr.io
-    repository: spiffe/tornjak
+    repository: spiffe/tornjak-be
     pullPolicy: IfNotPresent
-    # Overrides the image tag whose default is the chart appVersion.
-    # TODO we should use a specific Tornjak version instead of 'latest'
-    version: "latest"
+    # -- Overrides the image tag whose default is the chart appVersion.
+    version: "v1.0.2"
+  service:
+    type: ClusterIP
+    port: 10000
+    annotations: {}
   config:
-    # Front-end specific configuration:
-    frontend:
-      # apiServerURL - URL of the Tornjak back-end
-      apiServerURL: "http://localhost:10000"  # 👈 Use it for minikube or kind
-     # Back-end specific configuration
-    backend:
-      # dataStore - persistent DB for storing Tornjak specific information
-      dataStore:
-        driver: "sqlite3"
-        file: "/run/spire/data/tornjak.sqlite3"
+    # -- persistent DB for storing Tornjak specific information
+    dataStore:
```

Comment:
I seen a new PR changing this to `persistence` so we might have to change this later here as well if we decide to continue with that PR, which I think we agreed to do at a later stage combining with other breaking changes.

Reply:
I had a conversation with @kfox1111 earlier in this thread about this. Since this Tornjak DB can be deployed in any environment, not necessarily on pvc, he was OK with keeping the name as is.

Reply:
This one is actually the settings in the config file named dataStore. So I think this one makes sense to leave as is. The other one was entirely settings related to the pvc, which is named persistence in most other charts.